README.md

```diff
@@ -1,20 +1,37 @@
 
-This template repository equips you with the default initial files required for a Ministry of Justice GitHub repository.
+This template repository equips you with the default initial files required for an Analytical Platform Image Build Template GitHub repository.
+
+This repository is managed in Terraform [here](https://github.com/ministryofjustice/data-platform/blob/main/terraform/github/analytical-platform-repositories.tf#L286).
```

Comment:
This link will change as that file evolves, maybe just link to the file instead of a specific line